.azure-pipelines/azure-pipelines-data-container.yml

```diff
@@ -29,13 +47,13 @@ steps:
   inputs:
     SourceFolder: $(Build.SourcesDirectory)
     contents: .git/**
-    targetFolder: $(Build.SourcesDirectory)/scripts/datacontainer
+    targetFolder: $(Build.SourcesDirectory)/${{ parameters.PathToDockerfile }}
 
 - task: Docker@2
   displayName: Build and Push
   inputs:
     command: buildAndPush
-    Dockerfile: scripts/datacontainer/Dockerfile
+    Dockerfile: ${{ parameters.PathToDockerfile }}/Dockerfile
     repository: $(CONTAINER_REGISTRY_REPOSITORY_NAME)
```

Comment:
This variable is currently defined in the pipeline variables but if we want to publish a unique windows and ubuntu image we will need to give them different names. We also need confirmation about which container registry to publish this new one to before we go and try running this.